peoplepage.html

```diff
@@ -5,22 +5,38 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Artists</title>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.0/css/bootstrap.min.css">
     <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+        * {
+            font-family: 'Poppins', sans-serif;
+            margin: 0;
+            padding: 0;
+        }
+        .artist{
+            margin-left: 65px;
+        }
         .artist-card {
             width: 200px;
             margin: 17px;
-            margin-bottom: 20px;
+            margin-bottom: 10px;
             text-align: center;
+            border-radius: 10px;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+            transition: transform 0.3s ease;
+        }
+        .artist-card:hover {
+            transform: translateY(-5px);
         }
 
         .artist-card img {
-            width: 150px;
-            height: 200px;
+            width: 100%;
+            height: 80%;
             object-fit: cover;
             margin: 0px;
             margin-bottom: 10px;
             border-radius: 9px;
+            margin-top: 10px;
         }
 
         nav{
```

Comment:
change  from nav to header to remove the uncovered  part on the left of the navbar

Reply:
done